Source/Tests/FEXLoader.cpp

```diff
@@ -667,6 +673,7 @@ int main(int argc, char **argv, char **const envp) {
 
   if (AOTIRCapture() || AOTIRGenerate()) {
     FEXCore::Context::FinalizeAOTIRCache(CTX);
+    // Rename the temporary file to atomically update the file
```

Comment:
Stray comment? Or maybe this should go above `FinalizeAOTIRCache`?

Reply:
Resolved. Comment was meant to be above in the `SetAOTIRRenamer` lambda.